pkg/spegel/spegel.go

```diff
@@ -188,12 +187,12 @@ func (c *Config) Start(ctx context.Context, nodeConfig *config.Node) error {
 		libp2p.Identity(p2pKey),
 		libp2p.Peerstore(ps),
 		libp2p.PrivateNetwork(c.PSK),
-		libp2p.PrometheusRegisterer(metrics.DefaultRegisterer),
```

Comment:
I think we need to keep this here to expose the libp2p metrics.

Reply:
I have added this in by default in the router.

https://github.com/XenitAB/spegel/blob/d84d7710aa733280f4923d5b8ee96d624c54fd0a/pkg/routing/p2p.go#L70